distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1335,6 +1343,9 @@ public void testFederatedDownloadProductToCacheOnlyCacheDisabled() throws Except
                 .body(is("Caching of products is not enabled."));
         // @formatter:on
 
+        // Wait 1 seconds so we can verify that the product never gets written to the cache.
```

Comment:
Instead of sleeping and checking for the file existence, maybe we should wait for the CometD event that says that the download is complete, start another download and make sure the file is downloaded again, i.e., still receiving CometD events?